ci/prepare.sh

```diff
@@ -34,4 +38,3 @@ echo $CLI_VERSION > PHAR_BUILD_VERSION
 
 mysql -e 'CREATE DATABASE wp_cli_test;' -uroot
 mysql -e 'GRANT ALL PRIVILEGES ON wp_cli_test.* TO "wp_cli_test"@"localhost" IDENTIFIED BY "password1"' -uroot
-mysql -e 'GRANT ALL PRIVILEGES ON wp_cli_test_scaffold.* TO "wp_cli_test"@"localhost" IDENTIFIED BY "password1"' -uroot
```

Comment:
Not sure but I think we might need to keep the `GRANT ALL PRIVILEGES ON wp_cli_test_scaffold.*`line here.

Reply:
Good catch, re-added that one.